src/control/tools/StrokeHandler.cpp

```diff
@@ -90,13 +142,14 @@ auto StrokeHandler::onMotionNotifyEvent(const PositionInputData& pos) -> bool {
 
         view.drawStroke(crMask, stroke, 0, 1, true, true);
     } else {
-        if (pointCount > 0) {
-            Point prevPoint(stroke->getPoint(pointCount - 1));
+        int pointCount = stroke->getPointCount();
+        if (pointCount > 1) {
```

Comment:
Since return type of `getPointCount` should be `size_t` use auto here, so fixing the function will not introduce a casting issue.
```suggestion
        if (auto const pointCount = stroke->getPointCount(); pointCount > 1) {
```